suggestionBox/suggestionBox-war/src/main/java/org/silverpeas/components/suggestionbox/control/SuggestionBoxWebController.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.silverpeas.components.suggestionbox.control;
 
+import org.silverpeas.core.contribution.model.CoreContributionType;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.contribution.model.CoreContributionType'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)